### ISSUE TITLE
Add caching header fields as parameters

### ISSUE THREAD

**Issue by Robbilie** [esi/esi-issues#456]
# Feature request
just like for X-User-Agent (header) and token (query) i'd like to request `if-none-match` and possibly `if-modified-since` to be added.

### Route
all

### Use case
use swagger-client to pass etags / timestamps (js client ie doesnt allow to set headers manually so this is the only way)

**Comment by aquarhead** [comment]
Could be integrated with ETag support? @a-tal

**Comment by a-tal** [comment]
yeah. I'd be ok with an `etag` qsarg to function as `if-none-match`

**Comment by joaomlneto** [comment]
This has been implemented (a while ago): https://developers.eveonline.com/docs/services/esi/best-practices/#caching
